tools/cd_scripts/e2e_test.sh

```diff
@@ -107,6 +107,7 @@ cd gcsfuse
 git checkout $(sed -n 2p ~/details.txt) |& tee -a ~/logs.txt
 
 #run tests with testbucket flag
+set +e
```

Comment:
with this change, would the problem be solved if FAIL didn't some in logs.txt ?

Reply:
Changed the if condition to use exit status of the previous command (e2e test)